src/src/AdminBundle/Tests/Controller/EventTypeControllerTest.php

```diff
@@ -0,0 +1,9 @@
+<?php
```

Comment:
Ak by sme testy nahodou aj chceli pisat tak maju ist do `src/tests/AdminBundle/Controller/` a nie sem

Reply:
Ono mi to samo vygenerovalo. Mam s tym nieco spravit?

Reply:
Zmazať file alebo ho presunúť. Tu ho totiž phpunite nenájde. Co ti tento súbor vygenerovalo?

Reply:
"Ono mi to samo vygenerovalo."

Reply:
"Co ti tento súbor vygenerovalo?"

Reply:
stale plati aby si ho bud zmazala alebo presunula.

Reply:
konkretne prikaz na vygenerovanie controllera. Nevytvaram ho totiz rucne. Oki, zmazem ho.

Reply:
" prikaz na vygenerovanie controllera. " dakujem :-) na toto som sa pytal (teda este keby skonkretizujes ako sa ten prikaz vola tak by ma to potesilo

Reply:
Lebo sa mi nezda, ze testy maju ist do fileu kde su controllery. Nie?

Reply:
ak myslis cestu ktoru som pisal ja, tak to je priecinok, nie subor a pozri si ako ta cesta zacina.